examples/different/problem_statement/problem.sv.md

```diff
@@ -0,0 +1,9 @@
+Skriv ett program som beräknar differensen mellan icke-negativa heltal.
+
+## Indata
+Indatan består av flera testfall (minst \(1\) och som mest \(40\)), ett per rad.
```

Comment:
I am not sure I am overly fond of the \\( and \\) syntax for math mode in markdown.